client/actions/assignments/ui.js

```diff
@@ -30,6 +30,26 @@ const loadAssignments = (
     return dispatch(queryAndSetAssignmentListGroups(filterByState));
 };
 
+const myAssignments = (
```

Comment:
This action is redundant considering all it does is call `queryAndGetMyAssignments`. Consider replacing all usages of `myAssignments` with `queryAndGetMyAssignments`